endToEndTests/test/queries/complexQuery.json

```
@@ -0,0 +1,115 @@
+{
+  "testCaseName": "Query that crashes SILO",
```

Hopefully not anymore ;)
```suggestion
  "testCaseName": "some complex query",
```